src/Editor.tsx

```diff
@@ -121,7 +123,7 @@ const Editor = ({
           <ListMaxIndentLevelPlugin maxDepth={listMaxIndent} />
           <LinkPlugin />
           <ClickableLinkPlugin />
-          <CharacterStylesPopupPlugin />
+          {_isEditable && <CharacterStylesPopupPlugin />}
```

Comment:
The popup would appear and enable formatting changes even when disabled was set to false.